src/sniffer/pkg/ipresolver/procfs_resolver.go

```diff
@@ -0,0 +1,206 @@
+package ipresolver
+
+import (
+	"errors"
+	"github.com/otterize/network-mapper/src/sniffer/pkg/utils"
+	"github.com/sirupsen/logrus"
+	"sync"
+)
+
+type ProcessMonitor struct {
```

Comment:
We better extract that to another package to keep "one type per file" rule

Reply:
They're highly coupled at the moment... but I guess it's a manner of taste :)

Reply:
Definitly taste and IDE crying